io/net/bonding.py

```diff
@@ -96,6 +96,10 @@ def setUp(self):
                                                 default=False)
         self.peer_wait_time = self.params.get("peer_wait_time", default=5)
         self.sleep_time = int(self.params.get("sleep_time", default=5))
+        self.ib = 'N'
```

Comment:
This can be False / True, so that the checking later can be
`if self.ib` instead of `if self.ib == 'Y'`

Reply:
ok can be done.